PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -53,6 +52,10 @@
     <FilesToCopy Include="$(SourcesRoot)\SonarRunner.Shim\sonar-runner.zip" />
     <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.Integration\bin\$(Configuration)\TeamBuild.SonarQube.Integration.dll" />
     <FilesToCopy Include="$(SourcesRoot)\SonarQube.MSBuild.Tasks\bin\$(Configuration)\SonarQube.Integration.Tasks.dll" />
+    <ProjectReference Include="..\..\V0_9UpgradeMessageExe\V0_9UpgradeMessageExe.csproj">
+      <Project>{d35af999-a91e-4de2-93d7-858dfb4dd063}</Project>
+      <Name>V0_9UpgradeMessageExe</Name>
+    </ProjectReference>
```

Comment:
The packaging project didn't reference the shim project and so failed to build unless the shim project had already been built.